options.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 let page = document.getElementById('buttonDiv');
-const kButtonColors = ['#3aa757', '#e8453c', '#f9bb2d', '#4688f1'];
+const kButtonColors = ['#ffa7ae', '#ffc0c5', '#ffdadd', '#fff3f4'];
```

Comment:
can you comment what are these colors?

Reply:
pink, pink, pink and pink.

Reply:
can you comment in the code what are these colors?